### DISCUSSION TITLE
Ludusavi on steamdeck only pulls 18 games, though I have over 50 installed.  Everything is native on steam, nothing is through another launcher or installer.

### DISCUSSION THREAD

**Discussion by shadow9d9** [mtkennerly/ludusavi#311] [Q&A]
Any ideas?  Thanks!

**Comment by mtkennerly** [comment]
Hi! Could you give me some examples of games that aren't being detected? That would help to narrow down the issue.

**Reply by shadow9d9** [reply]
The game I most want to back up is Bonfire Peaks.  80% of my games aren't showing up.  Ghost Shore, Gamdec, ace attorney chronicles, the excavation of hob's barrow, dome keeper, elsinore, deliver the moon, cobalt core, dead cells, chained echoes, altered, anodyne 2, beacon piones, astlibra, backpack hero., bayonetta are just a few.

**Reply by mtkennerly** [reply]
Ludusavi ultimately gets info about save locations from PCGamingWiki. The main issue here is that PCGW only has Windows info for some of these games. For example:

* [Bonfire Peaks](https://www.pcgamingwiki.com/wiki/Bonfire_Peaks#Game_data)
* [Bayonetta](https://www.pcgamingwiki.com/wiki/Bayonetta#Game_data)
* [Dome Keeper](https://www.pcgamingwiki.com/wiki/Dome_Keeper#Game_data)
* Ghost Shore - PCGW doesn't have an article with this name, but if you mean [Ghost on the Shore](https://www.pcgamingwiki.com/wiki/Ghost_on_the_Shore#Game_data), then it has the same issue

For those, if you or someone else were able to add the Linux save info to the wiki, then Ludusavi would automatically incorporate the new information within 24 hours.

On the other hand, for [Dead Cells](https://www.pcgamingwiki.com/wiki/Dead_Cells#Game_data), PCGW does have the Linux and Steam info, so that one ought to work. Could you double check if the info on the wiki is accurate for your system? If it is, then please try this test:

* Close Ludusavi
* Open a console: Steam Menu -> Power -> Switch to Desktop -> Application Launcher (lower left corner) -> System -> Konsole
* In the console, run this command:
  ```
  flatpak run --env RUST_LOG=ludusavi=trace com.github.mtkennerly.ludusavi backup --preview "Dead Cells"
  ```
* Send me the `ludusavi_rCURRENT.log` file from `~/.var/app/com.github.mtkennerly.ludusavi/config/ludusavi`

**Comment by spidy2356** [comment]
lududsavi can't detect my few games, for example call of Duty 4: Modern Warfare (2007). Why is that so?
<img width="250" height="139" alt="image" src="https://github.com/user-attachments/assets/43f6b726-38f9-4f96-83a6-2ab806933e08" />
<img width="1016" height="383" alt="image" src="https://github.com/user-attachments/assets/cd49120e-3449-481f-88b6-6a6482759bef" />
Also, Far Cry 3 and Red Dead Redemption are missing here ...is it due to the fact that I haven't played them yet??

**Reply by mtkennerly** [reply]
For Call of Duty, Ludusavi is expecting the title to be `Call of Duty 4: Modern Warfare` (matching the [PCGamingWiki article](https://www.pcgamingwiki.com/wiki/Call_of_Duty_4:_Modern_Warfare)), but you have `Call of Duty 4: Modern Warfare (2007)`, which doesn't match. There are two ways to solve that; you can pick whichever solution is more convenient for you:

1. In Ludusavi, create a custom game titled `Call of Duty 4: Modern Warfare (2007)` and set it as an alias of `Call of Duty 4: Modern Warfare`.
3. In Playnite, right click on the game, select `Ludusavi: Look up with another title`, and then select `Call of Duty 4: Modern Warfare` from the list.

For Far Cry 3 and Red Dead Redemption, yes, most likely the reason is that you haven't played them yet. If you haven't played them, then there wouldn't be any save data on your system for Ludusavi to find. Ludusavi will only list the games with save data on the system, not all games that you own or are installed but never played.
